README.md

```diff
@@ -946,7 +947,7 @@ Table of Contents
 
 ## Other Free Resources
 
-  * [education.github.com](https://education.github.com/pack) — Collection of free services for students. Registration required
+  * [education.github.com](https://education.github.com/pack) — Collection of free services for students. Registration required,see [mirror page](https://gh.fastio.me/).
```

Comment:
I'm ok just linking to the primary github.com site, it's always up to date and if it doesnt work - neither does any of the offers make any sense.  So why link to a mirror?

Reply:
I guess because Github is not available in China